src/renderer/App.js

```diff
@@ -299,127 +299,76 @@ export default Vue.extend({
     handleYoutubeLink: function (href) {
       const v = this
 
```

Comment:
Since you're using an arrow function in the callback, you can get rid of the `v` identifier and replace it with `this`.
Right now, there's a mixture of the two.